### PR TITLE
Feat/5 upload resume

### LINKED ISSUE #5: Implement UI to upload a resume
### Describe why the project needs this task
Every applicant should be able to upload his resume when applying to a in-house job posting in order to be considered a candidate. Without a UI to upload a resume this is not possible.

### Related user stories
* S10

### Describe the solution to the problem
An applicant will have a UI that lets the select a file (from their computer or from a cloud account).

### PR DESCRIPTION
**Pull request by sulemanmalik**
closes #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,13 +5,15 @@
   "dependencies": {
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
+    "@storybook/addon-actions": "^4.1.12",
```
Comment (erickzhao) [inline package.json]: don't need storybook dependencies

File: src/App.tsx [hunk 2]
```diff
@@ -19,6 +22,7 @@ class App extends Component {
       <Route path="/" component={Dashboard} exact />
       <Route path="/login" component={Login} />
       <Route path="/register" component={Register} />
+      <Route path="/resume-upload" component={ResumeUpload} />
```
Comment (erickzhao) [inline src/App.tsx]: we could probably just use a `/resume/` route

File: src/Features/ResumeUpload/Upload.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        }, 2000);
+      }}
+      render={({ submitForm }) => (
+        <Form>
+          <Field component={SimpleFileUpload} name="file" />
+          <br />
+          <Button variant="contained" color="primary" onClick={submitForm}>
+            Submit
+          </Button>
+          <br />
```
Comment (erickzhao) [inline src/Features/ResumeUpload/Upload.tsx]: No need for a `<br/>` here. If you need to add padding, do it via CSS.

**Review by erickzhao — CHANGES_REQUESTED** [review]
A few nits. Functionality good.

File: package.json (hunk 1 above)
Comment (sulemanmalik) [reply package.json]: removed

File: src/Features/ResumeUpload/Upload.tsx (hunk 3 above)
Comment (sulemanmalik) [reply src/Features/ResumeUpload/Upload.tsx]: done

**Review by erickzhao — APPROVED** [review]

**Pull request merged**